auth/jwt_test.go

```diff
@@ -47,48 +50,134 @@ func (suite *JWTAuthenticatorTestSuite) createWrongToken(method jwt.SigningMetho
 	return token.SignedString([]byte(config.GetString("auth.jwt.secret")))
 }
 
-func (suite *JWTAuthenticatorTestSuite) TestAuthenticate() {
-	user := &TestUser{}
-	tokenAuthenticator := &JWTAuthenticator{}
+func (suite *JWTAuthenticatorTestSuite) TestGenerateToken() {
 	token, err := GenerateToken("johndoe@example.org")
 	suite.Nil(err)
-	suite.Nil(tokenAuthenticator.Authenticate(suite.createRequest(token), user))
-	suite.Equal("Admin", user.Name)
+	suite.Nil(suite.tokenAuthenticator.Authenticate(suite.createRequest(token), suite.user))
+	suite.Equal("Admin", suite.user.Name)
+}
+
+func (suite *JWTAuthenticatorTestSuite) TestGenerateTokenValidates() {
+	token, err := GenerateToken("johndoe@example.org")
+	suite.Nil(err)
+	claims := jwt.MapClaims{}
+	_, err = jwt.ParseWithClaims(token, claims, func(token *jwt.Token) (interface{}, error) {
+		return []byte(config.GetString("auth.jwt.secret")), nil
+	})
+	suite.Nil(err)
+}
+
+func (suite *JWTAuthenticatorTestSuite) TestGenerateTokenHasClaims() {
+	token, err := GenerateToken("johndoe@example.org")
+	suite.Nil(err)
+	claims := jwt.MapClaims{}
+	_, err = jwt.ParseWithClaims(token, claims, func(token *jwt.Token) (interface{}, error) {
+		return []byte(config.GetString("auth.jwt.secret")), nil
+	})
```

Comment:
Should check signing method as well
```suggestion
	_, err = jwt.ParseWithClaims(token, claims, func(token *jwt.Token) (interface{}, error) {
		if _, ok := token.Method.(*jwt.SigningMethodHMAC); !ok {
			return nil, fmt.Errorf("Unexpected signing method: %v", token.Header["alg"])
		}
		return []byte(config.GetString("auth.jwt.secret")), nil
	})
```